README.md

```diff
@@ -97,7 +103,38 @@ Each lion should have:
 Create a new lion instance with the name `simba`
 
 ```ruby
-# code here
+class Animal
+  attr_accessor :name
+
+  def initialize name
+    @name = name
+  end
+  def greeter
+  puts  "Hello my name is #{name}"
```

Comment:
This will error out because `name` is a local variable that you haven't defined. You should use instance variable syntax.